src/pages/invoices/common/components/InvoiceTotals.tsx

```diff
@@ -72,8 +77,28 @@ export function InvoiceTotals(props: Props) {
     return false;
   };
 
+  useEffect(() => {
+    if (cardRef.current) {
+      const dashedBorders = cardRef.current.querySelectorAll('.border-dashed');
+
+      if (dashedBorders.length > 0) {
+        const lastElement = dashedBorders[
+          dashedBorders.length - 1
+        ] as HTMLElement;
+
+        lastElement.style.borderBottom = 'none';
+      }
+    }
+  }, [variables, company, resource]);
+
```

Comment:
Shouldn't Tailwind's `divide-y` work here?

Reply:
@beganovich Yeah, we can replace the solution with divide-y and adjust the colors of dividers. I replaced the solution with it. Let me know your thoughts.